.circleci/config.yml

```diff
@@ -2,6 +2,12 @@ version: 2.1
 
 orbs:
   slack: circleci/slack@3.4.2
+  swissknife: roopakv/swissknife@0.48.0
```

Comment:
This is not yet released @roopakv 
https://github.com/roopakv/orbs/tree/v0.47.0

Reply:
Orb.swissknife.dev :) it is released

Reply:
TIL how to check orb releases. 

https://circleci.com/orbs/registry/orb/roopakv/swissknife